charts/seqr/templates/migration.yaml

```diff
@@ -1,39 +0,0 @@
-{{- if .Values.run_seqr_database_migration }}
```

Comment:
this is out of date and unused, I think better to just re-write with the current `Job` setup that we're using for check-new-samples when we want to enable?